app/controllers/api/v1/contacts_controller.rb

```diff
@@ -0,0 +1,30 @@
+class Api::V1::ContactsController < Api::V1::BaseController
+  def index
+    render json: Contact.order(created_at: :desc)
+  end
+
+  def create
+    @contact = Contact.new(contact_params)
+
+    if @contact.save
+      redirect_to admin_contacts_url(@contact), notice: "投稿を作成しました。"
+    else
+      render :new
+    end
+  end
+
+  private
+
+    # Never trust parameters from the scary internet, only allow the white list through.
+    def contact_params
+      params.require(:contact).permit(
+        :corner,
+        :message,
+        :nickname,
+        :name,
+        :department,
+        :grade,
+        :readable
```

Comment:
__[RuboCop]__

Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.


<!-- Comment by [SideCI](https://sideci.com) -->